src/applications/check-in/components/pages/AppointmentDetails/AppointmentDetails.unit.spec.jsx

```diff
@@ -355,7 +391,7 @@ describe('check-in experience', () => {
             .exist;
         });
       });
-      describe('CVT pre-check-in appointment', () => {
+      xdescribe('CVT pre-check-in appointment', () => {
```

Comment:
what's going on with all these `xdescribe` changes? Does that do a skip?

Reply:
![Screenshot 2024-07-03 at 9 18 16 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/2982977/fd328e8d-f157-4444-803e-040a5bfca52e)
😂 that's one way to fix the tests, Sometimes I do that to isolate the one test that isn't working. I forgot to go back and remove those.